src/components/popup-menu/popup-menu.styles.js

```diff
@@ -0,0 +1,111 @@
+export default ({ palette, typography, mixins }) => ({
```

Comment:
Has this been visually tested in more than one browser? What about IE11?

Reply:
Tested in Chrome, Safari, Firefox, IE11